### PR TITLE
Enhance code copy file downloads

### PR DESCRIPTION
**Pull request by egkrinkel**
Related To: https://github.com/SallaApp/Store/pull/4188

What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* enhance

What is the current behaviour? (You can also link to an open issue here)

* https://app.clickup.com/t/864dv26pp

What is the new behaviour? (You can also link to the ticket here)

* https://app.clickup.com/t/864dv26pp

Does this PR introduce a breaking change?

* 

Screenshots (If appropriate)

* 
Screenshot 2023-02-01 at 3 32 05 AM
Screenshot 2023-02-01 at 3 31 12 AM

### CODE REVIEW

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

File: src/views/pages/thank-you.twig [hunk 1]
```diff
@@ -49,6 +49,16 @@
...
             {% for item in order.items %}
+                {% if item.has_files %}
+                    <div class="flex flex-col items-center pt-1 pb-10 justify-start rtl:space-x-reverse space-x-2">
+                        <span class="font-bold">{{ trans('pages.thank_you.find_files') }}</span>
+                    </div>
+                {% elseif item.has_codes %}
+                    <div class="flex flex-col items-center pt-1 pb-10 justify-start rtl:space-x-reverse space-x-2">
+                        <span class="font-bold">{{ trans('pages.thank_you.find_codes') }}</span>
+                    </div>
+                {% endif %}
```
Comment (jalmatari) [inline src/views/pages/thank-you.twig:60]: use : `item.files|length` & `item.codes|length`

**Review by jalmatari — DISMISSED** [review]

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/Store/pull/4188 has Merge conflict❌.

> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 Branch is up-to-date..✅

**Comment by SallaDev** [comment]
> The branch `enhance-code-copy-file-downloads` of https://github.com/SallaApp/theme-raed/pull/179 has been synced from master..✅

**Review by AIMSemo — APPROVED** [review]

**Pull request merged**